parm/soca/obs/config/adt_rads_all.yaml

```diff
@@ -17,6 +17,13 @@ obs operator:
   name: ADT
```

Comment:
so you removed the halo distribution, does the LETKF observer work without the halo distribution?

Reply:
It gets inserted dynamically in the script until the obs config files get jinjafied